<launch>

	<arg name="rviz" default="true" />

	<rosparam command="load" file="$(find map_relocalization)/config/config_livox0830_seaside1.yaml" />

	<!-- <param name="feature_extract_enable" type="bool" value="0"/>
	<param name="point_filter_num" type="int" value="3"/>
	<param name="max_iteration" type="int" value="3" />
	<param name="filter_size_surf" type="double" value="0.5" />
	<param name="filter_size_map" type="double" value="0.5" />
	<param name="cube_side_length" type="double" value="1000" />
	<param name="runtime_pos_log_enable" type="bool" value="0" /> -->
    <node pkg="map_relocalization" type="map_relocalization_node" name="localization_in_map" output="screen" /> 

	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find map_relocalization)/rviz/map_relocalization.rviz"/>
	<!-- <group if="$(arg rviz)">
	</group> -->

</launch>
